packages/@webex/internal-plugin-ai-assistant/src/ai-assistant.ts

```diff
@@ -0,0 +1,447 @@
+              message: concatenatedMessage,
+              requestId,
+              finished: false,
+              errorMessage,
+              errorCode,
+            });
+          } catch (decryptError) {
+            // If decryption fails, we still want to continue listening for more messages
+            this.trigger(streamEventName, {
+              message: concatenatedMessage,
+              requestId,
+              finished: false,
+              errorMessage: errorMessage || decryptError.message,
+              errorCode,
+            });
+          }
+        }
+      });
+
+      this.webex
+        .request({
+          service: AI_ASSISTANT_SERVICE_NAME,
+          resource,
+          method: 'POST',
+          contentType: 'application/json',
+          body: {clientRequestId: requestId, ...params},
+        })
+        .catch((error) => {
+          reject(error);
+        })
+        .then(({body}) => {
+          const {sessionId} = body;
+
+          resolve({
+            requestId,
+            sessionId,
+            streamEventName,
+          });
+          timer.start();
+        });
+    });
+  },
+
+  /**
+   * Common method to make AI assistant requests for meeting analysis
+   * @param {Object} options
+   * @param {string} options.contextResources array of context resources to include in the request
+   * @param {string} options.sessionId the session ID for subsequent requests, not required for the first request
+   * @param {string} options.encryptionKeyUrl the encryption key URL for this meeting summary
+   * @param {string} options.contentType the type of content ('action' or 'message')
+   * @param {string} options.contentValue the value to use (action name or message text)
+   * @param {Object} options.parameters optional parameters to include in the request (for action type only)
+   * @param {Object} options.assistant optional parameter to specify the assistant to use
+   * @returns {Promise<Object>} Resolves with an object containing the requestId, sessionId and streamEventName
+   */
+  async _makeMeetingRequest(options: MakeMeetingRequestOptions): Promise<RequestResponse> {
+    let value = options.contentValue;
+
+    if (options.contentType === 'message') {
+      value = await this._encryptData({
+        text: options.contentValue,
+        encryptionKeyUrl: options.encryptionKeyUrl,
+      });
+    }
+
+    const content: any = {
+      context: {
+        resources: options.contextResources,
+      },
+      encryptionKeyUrl: options.encryptionKeyUrl,
+      type: options.contentType,
+      value,
+    };
+
+    if (options.contentType === 'action' && options.parameters) {
+      content.parameters = options.parameters;
+    }
+
+    return this._request({
+      resource: options.sessionId ? `sessions/${options.sessionId}/messages` : 'sessions/messages',
+      dataPath: 'response.content',
+      params: {
+        async: 'chunked',
+        locale: 'en_US',
```

Comment:
should probably be a param that defaults to `en_US`? not sure how well AI assistant works with other languages.